### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by x5engine**
please add comments here and I do the changes

### CODE REVIEW

File: src/app/mycomments.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
   ListItemSecondaryAction,
   IconButton,
   withStyles,
+  Paper
```
Comment (friedger) [inline src/app/mycomments.js]: Not used in this file.

File: src/app/mycomments.js [hunk 2]
```diff
@@ -257,6 +258,8 @@ class MyComments extends React.Component {
       showUpdateDialog,
       currentVisibility,
       updating,
+      usersComments,
+      ownersComments
```
Comment (friedger) [inline src/app/mycomments.js]: Not used

File: src/components/model.js [hunk 3]
```diff
@@ -165,6 +165,20 @@ export const saveOwnerComment = async (commentText, currentComment) => {
   await updatedComment.save()
 }
 
+export const getUserComments = async (limit=10) => {
+  const comments = await UserComment.fetchList({
+    sort: '-order'
```
Comment (friedger) [inline src/components/model.js]: The model does not have this field. The methods are not currently used and can be removed.

File: src/components/search.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ import Img from 'gatsby-image'
 
 const styles = theme => ({
   margin: {
-    margin: theme.spacing(1),
+    margin: theme.spacing(1),//new syntax for spacing in mui
```
Comment (friedger) [inline src/components/search.js]: comment can be removed.

File: src/templates/appcodetails.js [hunk 5]
```diff
@@ -369,6 +369,7 @@ class AppDetails extends Component {
             object: this.props.data.apps.website,
           })
     await comment.save()
+    // await comment.insertOne()
```
Comment (friedger) [inline src/templates/appcodetails.js:373]: comment can be removed.

**Review by friedger — CHANGES_REQUESTED** [review]
Please remove public folder.

File: src/app/mycomments.js (hunk 1 above)
Comment (x5engine) [reply src/app/mycomments.js]: done

**Comment by x5engine** [comment]
> Please remove public folder.

I think the .ignore should not be sending the public folder each time we commit right?

File: src/app/mycomments.js (hunk 2 above)
Comment (x5engine) [reply src/app/mycomments.js]: done

File: src/components/model.js (hunk 3 above)
Comment (x5engine) [reply src/components/model.js]: done

File: src/components/search.js (hunk 4 above)
Comment (x5engine) [reply src/components/search.js]: cleaned up

**Review by friedger — APPROVED** [review]
Not all comments have been addressed, but I will merge this now. Thank you for the contribution!

**Pull request merged**